src/components/SingleEvent/EventParticipants.js

```diff
@@ -47,13 +56,41 @@ class EventParticipants extends Component {
       <Fragment>
         <H3>Participants - <Spots>{spots}</Spots></H3>
         <EventFilters handleSearch={handleSearch} />
+        {amAdmin? (
+          <SafeQuery
+                query={QRSupportedQuery}
+                variables={{ address: '1' }}
+              >
+                {result => {
+                  if (result.data.scanQRCodeSupported && result.data.scanQRCodeSupported.supported) {
+                    return(
+                      <SafeMutation mutation={ScanQRCode}>
+                        {scanQRCode => (
+                          <QRCodeContainer>
+                            <Button 
+                              onClick={ (() => {
+                              scanQRCode().then((result)=>{
+                                setSearchTerm(result.data.scanQRCode.address)
+                              })
+                            }) }>Scan QRCode</Button>
+                          </QRCodeContainer>
+                        )}  
+                      </SafeMutation>          
+                    )
+                  } else {
```

Comment:
I would rather we display nothing if QR code scanning is not supported.

Reply:
Fixed